### ISSUE TITLE
Change "Mates Controller" library name

### ISSUE THREAD

**Issue by BreadBoardMates** [arduino/library-registry#500]
I need help to change our library name from "Mates Controller" to "MatesController". The space is removed as per the warning.

Here's the repository link: https://github.com/BreadBoardMates/Arduino-Mates-Controller

The latest release is 1.0.1.

Thanks a lot!

**Comment by per1234** [comment]
The requested change is now in progress @BreadBoardMates .

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests and closes those issues/PRs when that is done. If you are subscribed to this issue or the linked PR https://github.com/arduino/library-registry/pull/501, you'll get a notification at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Comment by rsora** [comment]
Solved by #501

**Comment by BreadBoardMates** [comment]
Hi all,

Upon checking [this](http://downloads.arduino.cc/libraries/logs/github.com/BreadBoardMates/Arduino-Mates-Controller/), we got this result:

```
2021/09/24 01:16:38 Scraping https://github.com/BreadBoardMates/Arduino-Mates-Controller.git
2021/09/24 01:16:38 Checking out tag: v1.0.0
2021/09/24 01:16:39 Release Mates Controller:1.0.0 has wrong library name, should be MatesController
2021/09/24 01:16:39 Checking out tag: v1.0.1
2021/09/24 01:16:39 Release Mates Controller:1.0.1 has wrong library name, should be MatesController
2021/09/24 01:16:39 Checking out tag: v1.0.2
```

It didn't log anything else afterwards, neither a success nor a failure log. The v1.0.2 is the latest release with the correct updated name. What could be the problem?

Just a bit of an idea regarding what we did recently, we did an updated v1.0.1 release which reverted to the old name so that Arduino can add it to the library manager while waiting for the name change request to reflect. Then we released v1.0.2 with the updated name when we noticed that the new name is being checked.

Thank you for your assistance!
